requirements.txt

```diff
@@ -20,28 +20,28 @@ certifi==2018.10.15
 cffi==1.11.5
 chardet==3.0.4
 Click==7.0
-contextlib2==0.4.0
-cryptography==2.3
+contextlib2==0.5.5
+cryptography==2.3.1
 docopt==0.4.0
 docutils==0.14
 Flask-Script==2.0.6
-future==0.17.0
+future==0.17.1
 idna==2.7
 inflection==0.3.1
 Jinja2==2.10
 jmespath==0.9.3
-mailchimp3==2.0.11
+mailchimp3==3.0.6
```

Comment:
We don't use mailchimp in the admin, but we should probably stick to v2.0.11 like we do elsewhere.